docs/pages/accounts/account-types.mdx

```diff
@@ -136,3 +136,9 @@ Access for members is managed through a role-based system. Users can have the _o
 | **Admin**     | Can control most settings on your account, including signing up for paid services, changing permissions of other users, and managing programmatic access. |
 | **Developer** | Can create new projects, make new builds, release updates, and manage credentials.                                                                        |
 | **Viewer**    | Can only view your projects through Expo Go but cannot modify your projects in any way.                                                                   |
+
+### Security activity
+
+Security activity is a list of changes that happened to an EAS user profile – your personal account. It includes, among others, change of password, email, and 2FA authentication setup.
+
+It can be found in the [EAS user profile settings](https://expo.dev/settings).
```

Comment:
```suggestion
It can be found under **Overview** > [**User settings**](https://expo.dev/settings).
```

Let's call the path out since "EAS user profile settings" is not something we use in dashboard.